backend/src/interfaces.ts

```diff
@@ -0,0 +1,36 @@
+export enum GameState {
+  IN_PROGRESS = 'in_progress',
+  FINISHED = 'finished'
+}
+
+// for destination, I'm assuming it will be a link
+// I'll change it if needed
+export interface ButtonCoordinates {
```

Comment:
could you pls clarify what you mean by x/y? or more generally the purpose of this interface :)

Reply:
nvm i read level i see i see im still a little confused by xy tho

Reply:
think it's meant to be where the co-ordinates of where the button will appear on the pano

Reply:
> think it's meant to be the co-ordinates of where the button will appear on the pano

Yeah that's what I meant but I'll rename it to be clearer

Reply:
thanks for clarifying! im still a little concerned abt what the purpose of these buttons is though?

Reply:
the buttons would be for navigation from one location to another, like in google street view

Reply:
i see, thanks for explaining - thankfully we actually won't need such buttons i don't think :) pannellum has a [hotspot](https://pannellum.org/documentation/examples/hot-spots/) feature that does exactly what we need. if we have time we might implement custom buttons but i dont think we need to. we could always just fork pannellum and make changes to that.

Reply:
I removed it 👍